src/healthapp/windows/personal_details.py

```diff
@@ -18,28 +18,34 @@ def get_content(self) -> toga.Box:
         header_box = toga.Box(style=Pack(direction=COLUMN, padding=(20, 20, 0)))
         main_box = toga.Box(style=Pack(direction=COLUMN, padding=(2, 2), background_color="#fbf5cc"))
         main_black_box = toga.Box(style=Pack(direction=COLUMN, padding=(0, 18, 18), background_color="black"))
-        footer_box = toga.Box(style=Pack(padding=20))
+        footer_box = toga.Box(style=Pack(padding=5))
 
         # label for personal details
-        header_label = toga.Label("Age, Height and Weight: ", style=Pack(font_size=20, padding=(5, 10)))
-        age_label = toga.Label("Age: ", style=Pack(font_size=15, padding=(0, 10)))
-        height_label = toga.Label("Height: ", style=Pack(font_size=15, padding=(0, 10)))
-        weight_label = toga.Label("Weight: ", style=Pack(font_size=15, padding=(0, 10)))
+        header_label = toga.Label("Age, Height, Weight and BMI: ", style=Pack(font_size=20, padding=(10, 5)))
+        age_label = toga.Label("Age: ", style=Pack(color = 'black', font_size=15, padding=(0, 15)))
+        height_label = toga.Label("Height: ", style=Pack(color = 'black', font_size=15, padding=(0, 15)))
+        weight_label = toga.Label("Weight: ", style=Pack(color = 'black', font_size=15, padding=(0, 15)))
+        bmi_label = toga.Label("BMI: ", style=Pack(color = 'black', font_size=15, padding=(0, 15)))
 
         # text input + button for physical analysis
-        self.age_input = toga.TextInput(placeholder='Enter your age', style=Pack(background_color="#fbf5cc"))
-        age_input_box = create_border(self.age_input, inner_color="#fbf5cc")
-
-        self.height_input = toga.TextInput(placeholder='cm', style=Pack(background_color="#fbf5cc"))
-        height_input_box = create_border(self.height_input, inner_color="#fbf5cc")
-
-        self.weight_input = toga.TextInput(placeholder='kg', style=Pack(background_color="#fbf5cc"))
-        weight_input_box = create_border(self.weight_input, inner_color="#fbf5cc")
-
-        submit_button = toga.Button('Submit', on_press=self.submit_handler, style=Pack(background_color="#fbf5cc", padding=(-3)))
+        self.age_input = toga.TextInput(placeholder='Enter your age', style=Pack(color = 'black', background_color="#fbf5cc"), value=self.app.user.age)
+        age_input_box = create_border(self.age_input, inner_color="#fbf5cc", padding=(2, 13, 0))
+
+        self.height_input = toga.TextInput(placeholder='cm', style=Pack(color = 'black', background_color="#fbf5cc"), value=self.app.user.height)
+        height_input_box = create_border(self.height_input, inner_color="#fbf5cc", padding=(2, 13, 0))
+
+        self.weight_input = toga.TextInput(placeholder='kg', style=Pack(color = 'black', background_color="#fbf5cc"), value=self.app.user.weight)
+        weight_input_box = create_border(self.weight_input, inner_color="#fbf5cc", padding=(2, 13, 0))
+        # BMI -------------------------------------------------------------------------
+        self.app.user.update_bmi()
```

Comment:
Future, could probably change the bmi update thing to be run after something is changed/entered in the input boxes

Reply:
Sounds good, will look at that in due time.

Reply:
Made BMI label update upon pressing submit button + notifying the user the BMI has been updated.